pkg/cmd/roachtest/inconsistency.go

```diff
@@ -85,6 +81,21 @@ func runInconsistency(ctx context.Context, t *test, c *cluster) {
 		}
 		return nil
 	})
+
+	// Give the cluster time to fully come together, then set an aggressive
+	// consistency check interval. This makes sure that the consistency check runs
+	// against all three nodes. If it targeted only two nodes, a random one would
+	// fatal - not what we want.
+	time.Sleep(20 * time.Second)
```

Comment:
I stand behind moving the cluster setting change to after the call for `waitForFullReplication()~

However I do not understand the 20s wait - isn't `waitForFullReplication()` sufficient to "wait for the cluster to come together"? 


Also, separately, a question: the comment says "this makes sure that the consistency check runs against all three nodes". However the cluster setting change is onyl effected against node 2. Given that cluster settings take a little while to propagate, it looks to me like node 2 is the one that's going to predominently fall over, as it will start its checks before the other two nodes get the setting update. Is that intentional?

Reply:
> However I do not understand the 20s wait - isn't waitForFullReplication() sufficient to "wait for the cluster to come together"?

You're right that this should not be necessary. If replicas are moved to the node, it is likely (though not guaranteed, but alas) that it is considered live by all. Removed.

> Also, separately, a question: the comment says "this makes sure that the consistency check runs against all three nodes". However the cluster setting change is onyl effected against node 2. Given that cluster settings take a little while to propagate, it looks to me like node 2 is the one that's going to predominently fall over, as it will start its checks before the other two nodes get the setting update. Is that intentional?

The consistency check runs against all members of the range. It does not matter which node it gets run on, other than that that node's view of liveness determines who is probed as part of the check. I am choosing n2 here since n1 is the one we expect to die; it may already be dead (though it usually isn't).